apps/condo/domains/billing/access/ValidateQRCodeService.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Generated by `createservice billing.ValidateQRCodeService --type mutations`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canValidateQRCode ({ authentication: { item: user } }) {
```

Comment:
Don't we need to add access to support here?

Reply:
as wee see it, support will not be able to call this mutation